app/assets/javascripts/setup/bootstrap.js

```diff
@@ -0,0 +1,76 @@
+(function (window) {
+    var dom = {
+        EXTERNAL_K8S_FQDN_GROUP: '.form-group-external-kubernetes',
+        EXTERNAL_K8S_FQDN_INPUT: '#settings_apiserver',
+        EXTERNAL_VELUM_FQDN_GROUP: ".form-group-external-velum",
+        EXTERNAL_VELUM_FQDN_INPUT: "#settings_dashboard_external_fqdn",
+        TRAILING_DOT_K8S_MSG: ".trailing-dot-k8s",
+        TRAILING_DOT_VELUM_MSG: ".trailing-dot-velum",
+        BOOTSTRAP_BTN: "#bootstrap"
+    };
```

Comment:
It might be nice to make the quoting (single v/s double) consistent in this structure.